tox.ini

```diff
@@ -62,6 +62,7 @@ setenv =
     coverage: _PYTEST_TOX_EXTRA_DEP=coverage-enable-subprocess
     coverage: COVERAGE_FILE={toxinidir}/.coverage
     coverage: COVERAGE_PROCESS_START={toxinidir}/.coveragerc
+    coverage: PYTEST_ADDOPTS=--junitxml=junit.xml -o junit_family=legacy
```

Comment:
JUnit isn't coverage, though... Also, is it possible to use long option names?

P.S. It'd be good not to generate this in local runs, but only CI.

Reply:
> It'd be good not to generate this in local runs, but only CI.


TBH I wouldn't care too much... we need however to add `junit.xml` to `.gitignore`. However, if it is simple to not generate this for local runs, we can go ahead, but I wouldn't complicate things too much if not.

Reply:
Fair. In my CIs, I have built-in ways of having this separation. Perhaps, one day I'll get to refactoring this one as well...